CoreUtilities/src/au/gov/asd/tac/constellation/utilities/visual/VisualChange.java

```diff
@@ -119,4 +119,22 @@ public int hashCode() {
     public int compareTo(VisualChange o) {
         return o == null ? -1 : id == o.id ? 0 : Integer.compare(order, o.order);
     }
+    
+    public int[] getRange() {
```

Comment:
Can you please add a comment for this function and what it is returning

Reply:
Comment added.